package.json

```diff
@@ -8,6 +8,7 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
+    "lint:fix" : "next lint --fix",
```

Comment:
please, add a new pre-commit git hook that runs this script. That way, we make sure that the code is _linted_ when it reaches the origin

Reply:
Done!